bpm/bonita-core/bonita-home-server/src/main/java/org/bonitasoft/engine/home/FolderMgr.java

```diff
@@ -30,8 +30,8 @@ private static Folder getConfFolder(final File bonitaHomeFolder) throws IOExcept
         return getFolder(getServerFolder(bonitaHomeFolder), "conf");
     }
 
-    private static Folder getTempFolder(final File bonitaHomeFolder) throws IOException {
-        final Folder tempFolder = getFolder(getServerFolder(bonitaHomeFolder), "temp");
+    private static Folder getTempFolder() throws IOException {
+        final Folder tempFolder = getFolder(new File(System.getProperty("java.io.tmpdir")), "bonita_tmp_" + getJvmName());
         if (!tempFolder.exists()) {
             tempFolder.createAsTemporaryFolder();
```

Comment:
I know this as nothing to do with this PR but we should be careful with this method because each time we call it it registers a shutdown hook that is a thread to be started at jvm shutdown